app/dialog/export/exportvideotab.cpp

```diff
@@ -49,28 +46,23 @@ ExportVideoTab::ExportVideoTab(ColorManager* color_manager, QWidget *parent) :
   outer_layout->addStretch();
 }
 
-int ExportVideoTab::SetFormat(ExportFormat::Format format)
-{
+int ExportVideoTab::SetFormat(ExportFormat::Format format) {
   format_ = format;
 
   QList<ExportCodec::Codec> vcodecs = ExportFormat::GetVideoCodecs(format);
   setEnabled(!vcodecs.isEmpty());
   codec_combobox()->clear();
-  foreach (ExportCodec::Codec vcodec, vcodecs) {
-    codec_combobox()->addItem(ExportCodec::GetCodecName(vcodec), vcodec);
-  }
+  foreach (ExportCodec::Codec vcodec, vcodecs) { codec_combobox()->addItem(ExportCodec::GetCodecName(vcodec), vcodec); }
```

Comment:
That's a really questionable formatting choice 🤔

Reply:
I guess maybe I shouldn't have just let clang-format run free, considering after checking the `.clang-format` file, I realize that it has set the column limit to 120, while `CONTRIBUTING.md` says 100.  I now tried reformatting with it set to 100, and I can see why that is probably not a good idea, so I will go through and fix the ones that are problematic if that is preferred.

Reply:
We are actually not sure how that `.clang-format` was created and how it got added, but it doesn't match the actual code style of 0.2, which follows Google's conventions for the most part. We will try to adjust the config to match the current code style.